spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
+    name { Faker::Name.name }
+    sequence(:email) { |i| "user-#{i}@email.com" }
```

Comment:
You can also use the `unique` method from Faker gem.
Example: `Faker::Internet.unique.email`.

Ref: https://github.com/faker-ruby/faker#ensuring-unique-values

Reply:
@violeta-p 
I have mixed feelings about the `unique` method in Faker gem, it doesn't have enough unique values to assign in tests in situations when a lot of unique values have to assigned in tests.

Reply:
I decided on using `sequence` after reading the factorybot documentation creating unique values. But first I tried `Faker::Internet.email`, didn't know I can use .unique inside the defined `Faker::Internet.email`. 
I am confused about `unique`  in the schema vs. `uniqueness` in the model. Where do they come from. Are they defined on the Object class? and how should one know which one to use and where?

Reply:
https://jacopretorius.net/2017/04/validate-uniqueness-in-rails.html